data-processor/src/test/groovy/io/micronaut/data/processor/sql/BuildUpdateSpec.groovy

```diff
@@ -174,4 +174,24 @@ interface PersonRepository extends CrudRepository<Person, Long> {
         def ex = thrown(RuntimeException)
         ex.message.contains("Update methods only support void or number based return types")
     }
+
+
+    void "test AutoGenerated update method"() {
+      given:
+      def repository = buildRepository('test.StudentRepository', """
+  import io.micronaut.data.jdbc.annotation.JdbcRepository;
+  import io.micronaut.data.model.query.builder.sql.Dialect;
+  import io.micronaut.data.tck.entities.Student;
+  
+  @JdbcRepository(dialect= Dialect.MYSQL)
+  interface StudentRepository extends CrudRepository<Student, Long> {
+  
+  }
+  """)
+      def method = repository.findPossibleMethods("update").findFirst().get()
+      def updateQuery = method.stringValue(Query).get()
+
+      expect:
+      updateQuery == 'UPDATE `student` SET `name`=?,`creation_time`=? WHERE (`id` = ?)'
```

Comment:
this creation_time is regular field whereas lastUpdatedTime is `GeneratedValue`.  lastUpdatedTime is not included in query.